app/views/subscriptions/index.slim

```diff
@@ -127,13 +127,25 @@
                 style='cursor: default'
                 v-tippy=true
               ) Active
-              = form_for(:subscription, method: :delete) do |f|
+
+              - if plan.paypal?
                 tippy content='Cancel your subscription. You will keep your benefits for the rest of this cycle.'
                   template v-slot='trigger'
-                    = f.button(icon('fas', 'times'),
-                      class: 'btn btn-lg btn-outline-danger', \
+                    = link_to( \
+                      icon('fas', 'times'), \
+                      api_webhooks_paypal_unsub_path, \
+                      class: "btn btn-lg btn-outline-danger", \
                       data: {disable_with: icon('fas', 'circle-notch', class: 'fa-spin').html_safe}, \
+                      method: :post \
                     )
+              - else
+                = form_for(:subscription, method: :delete) do |f|
+                  tippy content='Cancel your subscription. You will keep your benefits for the rest of this cycle.'
+                    template v-slot='trigger'
+                      = f.button(icon('fas', 'times'),
+                        class: 'btn btn-lg btn-outline-danger', \
+                        data: {disable_with: icon('fas', 'circle-notch', class: 'fa-spin').html_safe}, \
+                      )
```

Comment:
If the plan (as defined in this view) is a stripe plan, it will show the button to unsub from the stripe subscriptions. If somehow the user then still had an active paypal sub, they would be asked to cancel that on the next load. 

If you want I can make it so there is just one "cancel ALL subscriptions" regardless of where the external vendor is. I thought the approach I took was a good step towards segregating payment types.

Reply:
Oh gotcha! Sorry, I misread this `else` as being the PayPal case. I withdraw my comment then 😄